### PR TITLE
Edit profile design

### PR DESCRIPTION
**Pull request by RoCKnD79**
Due to complications I created a new branch instead of the edit_profile branch. 
Here I am pushing the changes that I was advised to make on the edit_profile PR.
Small changes + the addition of a navigation drawer which I hope we'll add in the profile activity during the second sprint.

### CODE REVIEW

File: app/src/main/java/com/github/orkest/View/MenuItem.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.github.orkest.View
+
+import androidx.compose.ui.graphics.vector.ImageVector
+
+data class MenuItem(
+    val id: String,
+    val title: String,
+    val contentDescription: String? = null, // for people with visual impairments
```
Comment (yasmineben510) [inline app/src/main/java/com/github/orkest/View/MenuItem.kt:8]: Nice thought!

**Review by yasmineben510 — APPROVED** [review]
Nice job! Great documentation and refactoring. 
Thanks for implementing the requested changes!
It would be nice to have tests for the menu drawer too, but it is okay, in the next sprint we will all focus on incrementing the code coverage either way.

File: app/src/main/java/com/github/orkest/View/EditProfileActivity.kt [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+                    text = "Cancel",
+                    modifier = Modifier.clickable { /* TODO */ },
+                    fontSize = 20.sp
+                )
+
+                // "save" clickable text (button)
+                Text(
+                    text = "Save",
+                    modifier = Modifier.clickable { /* TODO */ },
+                    fontSize = 20.sp
```
Comment (saraanej) [inline app/src/main/java/com/github/orkest/View/EditProfileActivity.kt:177]: Maybe creating constants for the font size would be nice for maintanability

File: app/src/main/java/com/github/orkest/View/EditProfileActivity.kt [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+
+/**
+ * Top part of the screen, containing:
+ * - "cancel" button on the top left to stop editing changes and returning to profile screen
+ * - "save" button saving changes to database, updating profile screen and returning to it
+ * - current profile picture
+ * - "edit picture" button to modify picture
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
```
Comment (saraanej) [inline app/src/main/java/com/github/orkest/View/EditProfileActivity.kt:155]: Comments are appreciated !

**Review by saraanej — COMMENTED** [review]
Thank you for your code. It was clear and well explained.

**Pull request merged**